server/config.py

```diff
@@ -3,7 +3,7 @@
 TEAM_NAME = os.environ.get('TEAM_NAME')
 DB_USER = os.environ.get('DB_USER')
 DB_PASSWORD = os.environ.get('DB_PASSWORD')
-SQLALCHEMY_DATABASE_URI = f'postgresql://{DB_USER}:{DB_PASSWORD}@localhost/crm_test'
+SQLALCHEMY_DATABASE_URI = f'postgresql://postgres:123456789@localhost/crm_test'
```

Comment:
You are hardcoding this

Reply:
yeah sorry was just to test. I didn't even mean to push this